src/app/containers/Figure/index.jsx

```diff
@@ -1,20 +1,44 @@
-import React, { useContext, Fragment } from 'react';
+import React, { useContext, Fragment, useState } from 'react';
 import { string, number, objectOf, any, bool } from 'prop-types';
 import Figure from '@bbc/psammead-figure';
 import Image, { AmpImg } from '@bbc/psammead-image';
 import ImagePlaceholder from '@bbc/psammead-image-placeholder';
 import LazyLoad from 'react-lazyload';
+import { Transition } from 'react-transition-group';
+import styled from 'styled-components';
 import Copyright from '../Copyright';
 import Caption from '../Caption';
 import { RequestContext } from '../../contexts/RequestContext';
 
 const LAZYLOAD_OFFSET = 250; // amount of pixels below the viewport to begin loading the image
+const FADE_IN_DURATION = 200; // number of miillisecconds to  apply the fade-in effect
 
-const renderImage = (imageToRender, lazyLoad) =>
+const transitionStyles = {
+  entering: { opacity: 0 },
+  entered: { opacity: 1 },
+  exiting: { opacity: 0 },
+  exited: { opacity: 0 },
+};
+
+const ImageWrapper = styled.div`
+  transition: ${() => `opacity ${FADE_IN_DURATION}ms ease-in-out`};
+  opacity: ${props => props.opacity || 0};
```

Comment:
```suggestion
  opacity: ${props => props.opacity || 1};
```
It'd be good for the default value to be to show the image, in case for any reason the prop `opacity` is not passed in.

Reply:
Checked this out locally, and discovered that this suggestion breaks the transition functionality for me.

Reply:
First load is @sareh's suggestion. Second load is original where fade in works for me.

![My Video Gif](https://user-images.githubusercontent.com/7690572/59275958-4472e500-8c55-11e9-9934-c22e1322246a.gif)

Reply:
@sareh, yeah setting it to 1 makes the transition look funny, I think we could go with 0 for now